pages/agent/gcloud.md.erb

```diff
@@ -0,0 +1,183 @@
+# Running Buildkite Agent on Google Cloud Platform
```

Comment:
@toolmantim mentioned earlier that the imperative sections made it sounds like they were sequential steps ("1. Run the agent, 2. Start a cluster, 3. Upload artifacts") instead of potential options. Do you think they should be imperative instead? Or maybe just this top-level heading should be imperative?